parameter/PathNavigator.cpp

```diff
@@ -94,19 +94,19 @@ std::string CPathNavigator::getCurrentPath() const
 {
     std::string strPath = "/";
 
-    if (!_uiCurrentIndex) {
+    if (!_currentIndex) {
 
         return strPath;
     }
 
-    uint32_t uiItem;
+    size_t item;
 
-    for (uiItem = 0; uiItem < _uiCurrentIndex - 1; uiItem++) {
+    for (item = 0; item < _currentIndex - 1; item++) {
```

Comment:
item is used on the next line + the algo is iterating over [0, _current], not [begin, end].